config-linux.md

```diff
@@ -769,13 +770,14 @@ The following rules on parameters MUST be applied:
 
 * If `closID` is set, and none of `l3CacheSchema`, `memBwSchema` or `schemata` are set, runtime MUST check if corresponding pre-configured directory `closID` is present in mounted `resctrl`. If such pre-configured directory `closID` exists, runtime MUST assign container to this `closID` and [generate an error](runtime.md#errors) if directory does not exist.
 
-* **`enableCMT`** *(boolean, OPTIONAL)* - specifies if Intel RDT CMT should be enabled:
-    * CMT (Cache Monitoring Technology) supports monitoring of the last-level cache (LLC) occupancy
-      for the container.
-
-* **`enableMBM`** *(boolean, OPTIONAL)* - specifies if Intel RDT MBM should be enabled:
-    * MBM (Memory Bandwidth Monitoring) supports monitoring of total and local memory bandwidth
```

Comment:
The history of this technically-breaking changes should be documented as a note?

Reply:
:+1:   thanks @AkihiroSuda, I think that makes sense (I was thinking something along the lines I described in the commit message(s)). Any suggestion where to put that note?

Reply:
Got it. I now added a NOTE to config-linux.md (in the IntelRdt section).

@AkihiroSuda PTAL